Pages/Modal/modal.css

```diff
@@ -0,0 +1,58 @@
+.modal {
+    background: var(--gray-lighter);
+    border-radius: 1rem;
+    min-width: 35rem;
+    max-width: 45rem;
+    z-index: 2;
+  }
```

Comment:
Modal should have a fixed position on the viewport.

Reply:
Do I need to fix the position now, cuz I think it will work with JS

Reply:
Okay then you can plan it later. For now hopefully it should be fine 🤞